module/VuFind/src/VuFind/Mailer/Factory.php

```diff
@@ -97,6 +97,6 @@ public function createService(ServiceLocatorInterface $sm)
         $config = $sm->get('VuFind\Config')->get('config');
 
         // Create service:
-        return new \VuFind\Mailer\Mailer($this->getTransport($config));
+        return new \VuFind\Mailer\Mailer($this->getTransport($config), $config);
```

Comment:
Is it worth injecting the whole configuration object for just one setting? Would it be more generic to pass an $options array with the override_from in it? I'm not completely opposed to passing $config, but since we've avoided it so far, it seems like it would be cleaner to maintain that separation. Simply adding a getOptions method to the factory would make this a lot more readable.